save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/ScoreCard.kt

```diff
@@ -35,7 +35,7 @@ external interface ProjectScoreCardProps : Props {
  * @return ReactElement
  */
 @Suppress("TOO_LONG_FUNCTION", "LongMethod")
-fun projectScoreCard() = FC<ProjectScoreCardProps> { props ->
+fun scoreCard() = FC<ScoreCardProps> { props ->
```

Comment:
Maybe we should start making these `val`s instead of `fun`s, especially if they don't accept parameters

Reply:
You mean something like this?
```
val card = FC<Props> { ... } ()
```

Reply:
Yes, then we won't need things like `val cardComponent = card()` before using it

Reply:
Talking about cards, we cannot do it because cards still have arguments
But there are some other components that could do so